### DISCUSSION TITLE
It is possible to create sphere packing in Neper. I mean I have a cubic RVE with 11 particles

### DISCUSSION THREAD

**Discussion by enghorbani** [neperfepx/neper#682] [Q&A] [answered]
I used this code in Neper to create a cubic RVE with 11 spherical particle:

neper -T -n 500 -domain "cube(1) xperiodic yperiodic zperiodic" -morpho "spheres = 11 radius = 0.1 volfrac = 0.5" -o Sphere_Packing

but it did not work. Could you please help me?

**Comment by rquey** [comment, marked as answer]
This is not something Neper can do, and not Neper syntax...
